### PR TITLE
Pause simulation

### PR DESCRIPTION
**Pull request by Jolijn2020**
Can pause/resume simulation using the button "View" -> "Pause Simulation", 
or by hitting the space bar.

### CODE REVIEW

**Review by S010MON — CHANGES_REQUESTED** [review]
This is a poor implementation.  You need to look into the JavaFX documentation more and see how the timing is actually run.

Docs here: https://docs.oracle.com/javase/8/javafx/api/javafx/animation/Timeline.html

See **GameEngine.java / Line 53** 
```java
timeline.play();
```
This is how the animation is played.  Use the stop method to stop it.

**Review by MatthewFoster02 — APPROVED** [review]
Works for me.

File: src/main/java/app/App.java [hunk 1]
```diff
@@ -20,6 +19,7 @@
     @Getter private StartMenu startMenu;
     @Getter private Stage stage;
     private Simulation simulation;
+    //private Timeline animation;
```
Comment (S010MON) [inline src/main/java/app/App.java:22]: Is this variable being used?

**Review by S010MON — APPROVED** [review]

File: src/main/java/app/App.java (hunk 1 above)
Comment (Jolijn2020) [reply src/main/java/app/App.java:22]: ah, sorry, forgot to delete from wrong attempt

**Comment by S010MON** [comment]
I would suggest that the `pause simulation` menu item is renamed `pause` and moved into the Edit.  It does not make sense to be in view, as it doesn't move the user to a new view

**Comment by Jolijn2020** [comment]
> I would suggest that the `pause simulation` menu item is renamed `pause` and moved into the Edit. It does not make sense to be in view, as it doesn't move the user to a new view

The issue specified it had to be under view, but I can change it to edit easily

**Review by MatthewFoster02 — APPROVED** [review]
New changes still work. Looks good.

**Review by S010MON — APPROVED** [review]

**Pull request merged**